pkg/apis/admissionregistration/v1beta1/defaults_test.go

```diff
@@ -35,7 +36,7 @@ func TestDefaultAdmissionWebhook(t *testing.T) {
 	exact := v1beta1.Exact
 	never := v1beta1.NeverReinvocationPolicy
 	thirty := int32(30)
-	allScopes := v1beta1.AllScopes
+	allScopes := v1.AllScopes
```

Comment:
should be able to revert to the v1beta1.AllScopes constant now
```suggestion
	allScopes := v1beta1.AllScopes
```